ods_ci/tests/Tests/0400__ods_dashboard/0410__ods_dashboard_projects/0410__ods_dashboard_projects.robot

```diff
@@ -15,9 +15,9 @@ Test Tags          Dashboard
 
 
 *** Variables ***
-${PRJ_TITLE}=   ODS-CI Common Prj
+${PRJ_TITLE}=   ods-ci-common-project
```

Comment:
Is this change intentional as part of this fix? It now matches what the `PRJ_RESOURCE_NAME` contains.

Reply:
Good point, looks I'm not suspecting enough

Reply:
yes intentional. was causing the following setup failure `ODS-CI Common Prj != ods-ci-common-project` as the project title used is now ods-ci-common-project

Reply:
I don't get it :slightly_smiling_face: This variable is to define the project title, no? So, what is causing it to be ods-ci-common-project? Isn't there some conflict with what is defined by the other variable I mentioned?

Reply:
hmmm yes good point. it looks like somewhere along the line ${PRJ_TITLE} is being changed ? Or not being checked correctly ?

Reply:
I commented a suggestion above what I think is the rootcause.